crates/sui-indexer-alt-consistent-store/src/db/mod.rs

```diff
@@ -213,6 +213,26 @@ impl Db {
         })
     }
 
+    /// Get the names of all column families in the database.
```

Comment:
Shouldn't expose this here -- `Db` is unaware of the column families that get opened on it. This is something that belongs on the `Store` abstraction, which accepts a `Schema` implementation which dictates which column families get opened.